package.json

```diff
@@ -2,7 +2,7 @@
   "name": "omnitone",
   "version": "0.2.0",
   "description": "Spatial Audio Decoder in Web Audio API",
-  "main": "src/main.js",
+  "main": "build/omnitone.js",
```

Comment:
No preference for me. I've seen other modules in the past use the unminified version, possibly for debugging purposes. `three.js` uses the unminified version [here](https://github.com/mrdoob/three.js/blob/dev/package.json#L5)